components/avatar/avatar.html

```diff
@@ -0,0 +1,127 @@
+          <ul class="right-nav">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Documentation</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        <div class="doc-container">
+          <div class="component-menu">
+            <div class="titles"><strong>Components</strong> 
+
+            </div>
+            <div class="list-items">
+              <ul>
+                <li><a href="#">Alert</a></li>
+                <li><a href="#">Avatar</a></li>
+                <li><a href="#">Badge</a></li>
+                <li><a href="#">Button</a></li>
+                <li><a href="#">Card</a></li>
+                <li><a href="#">Image</a></li>
+                <li><a href="#">Input</a></li>
+                <li><a href="#">Text Utilities</a></li>
+                <li><a href="#">List</a></li>
+                <li><a href="#">Navigation</a></li>
+                <li><a href="#">Modal</a></li>
+                <li><a href="#">Rating</a></li>
+                <li><a href="#">Tost</a></li>
+                <li><a href="#">Grid</a></li>
+                <li><a href="#">Slider</a></li>
+              
+              </ul>
+            </div>
+          </div>
+          <div class="doc-content">
+            <div class="heading">Avatar</div>
+            <div class="intro-line">
+                In simple words, Avatar is the profile picture we put on our profile information page, navigation bar, and so on. 
+            </div>
+            <div class="divider"></div>
+            <div class="sub-heading">
+              <strong>Image Avatar</strong>
+            </div>
+            <div class="sub-head-content">
+               Image avatar can be created by using <code>img</code>, <code>img</code>, <code>src</code>, <code>srcSet</code> to the component.
+            </div>
+            <div class="practical avatar-box">
+                    <img loading="lazy"src="https://images.unsplash.com/photo-1531427186611-ecfd6d936c79?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NHx8YXZhdGFyJTIwcHJvZmlsZXxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60" alt="image 1" class="avatar" >
+                
+                
+                <!-- <div class="alert error">
+                  <i class='bx bx-error-circle'></i>
+                  This is an error alert — check it out!
+                </div>
+                <div class="alert warning">
+                  
+                  <i class='bx bx-error'></i>
+                  This is a warning alert  — check it out!
+              
+                </div>
+                <div class="alert info">
+                  
+                  <i class='bx bx-info-circle'></i>
+                  This is an info alert — check it out!
+                
+                </div>
+                <div class="alert success">
+                  
+                  <i class='bx bx-check-circle'></i>
+                  This is a success alert — check it out!
```

Comment:
If this is unnecessary you should remove it

Reply:
Removed